src/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -325,8 +325,12 @@ convert_expr_to_smt(const binary_relation_exprt &binary_relation)
 
 static smt_termt convert_expr_to_smt(const plus_exprt &plus)
 {
-  UNIMPLEMENTED_FEATURE(
```

Comment:
⛏️ I would prefer it if commits could be titled such that the title encompasses the entirety of what they change. The fact that you effectively have "Also add conversion of `plus_exprt` to `smt_bit_vector_theoryt::addition`" in the message body is kind of a red flag which suggests that you have combined unrelated changes together (even if one change depends of the other).